include/wil/winrt.h

```diff
@@ -46,6 +46,12 @@ namespace std
 #pragma warning(pop)
 #endif
 #endif
+#if defined(WIL_ENABLE_EXCEPTIONS) && defined(__has_include)
+#if __has_include(<vector>)
+#define __WI_HAS_STD_VECTOR 1

```

Comment:
Is this good? Is it OK to assume that if you have vector and exceptions turned on, you're OK with bringing in here?

Reply:
I think so